libr/anal/fcn.c

```diff
@@ -302,12 +302,7 @@ R_API RAnalFunction *r_anal_fcn_new() {
 }
 
 R_API RList *r_anal_fcn_list_new() {
-	RList *list = r_list_new ();
-	if (!list) {
-		return NULL;
-	}
-	list->free = &r_anal_fcn_free;
-	return list;
+	return r_list_new ();
```

Comment:
thats not the same. should be `r_list_newf (r_anal_fcn_free);` this patch should be leaking more

Reply:
please see this commit message https://github.com/radare/radare2/pull/11674/commits/33815683e612bacb0e7aee17ebf0aab105ab7f83

Reply:
?

Reply:
Ok I got it. Fixing it.